index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head> 
+<title>Portfolio</title>
+<link rel="stylesheet" href="styles.css">
+</head>
+<nav id="navbar">
+  <ul id="nav-list">
+    <li>
+      <a href="#welcome-section">About</a>
+    </li>
+    <li>
+      <a href="#projects">Work</a>
+    </li>
+    <li>
+      <a href="#contact">Contact</a>
+    </li>
+  </ul>
+</nav>
+
+<body>
+  <section id="welcome-section" class="welcome-section">
+    <h1>Hey I am Mershark</h1>
+    <p>a web developer</p>
+  <section>
```

Comment:
I think your section tag doesn't have a closing tag. This can be done by putting "/" in front of the closing tag 👍